jbrowse/src/client/JBrowse/Browser/plugins/ExtendedVariantPlugin/ExtendedVariantDisplay/model.js

```diff
@@ -25,14 +25,16 @@ export default jbrowse => {
 
     .actions(self => ({
       selectFeature(feature) {
+        const widgetDisplayInfo = getContainingTrack(self).configuration.metadata.value
```

Comment:
and then just update this to look for the nested property (in a null-safe way)

Reply:
Updated to grab from new location in metadata. Null behavior handling maintained, can test by removing metadata from minimalSession and opening a track.